metrics/prometheus/prometheus_test.go

```diff
@@ -156,6 +156,40 @@ func TestRequestMetric(t *testing.T) {
 		})
 }
 
+func TestDebugRequestMetric(t *testing.T) {
+	testCases := []struct {
+		description                      string
+		givenDebugEnabledFlag            bool
+		givenAccountDebugMetricsDisabled bool
+		expectedAccountDebugCount        float64
+		expectedDebugCount               float64
+	}{
+		{
+			description:                      "Boolean flags should result in both metrics being updated",
+			givenDebugEnabledFlag:            true,
```

Comment:
Please add a test case where debug is enabled and account debug is disabled and another test case where debug is disabled and account debug is enabled